docker.go

```diff
@@ -158,6 +165,10 @@ func (r DockerContainerRun) Execute(imageID string) (Container, error) {
 		}
 	}
 
+	if r.bindport != "" {
+		r.env["PORT"] = r.bindport
+	}
```

Comment:
After we do the logic to set the --env flag, we can set the $PORT back to whatever r.bindport is